UnityProject/Assets/Scripts/Items/Implants/Organs/Stomach.cs

```diff
@@ -47,22 +47,15 @@ public override void ImplantPeriodicUpdate()
 
 			if (AllFat)
 			{
-				StartCoroutine(DelayAddFat());
+				var Added = Spawn.ServerPrefab(BodyFatToInstantiate.gameObject).GameObject.GetComponent<BodyFat>();
+				BodyFats.Add(Added);
+				RelatedPart.OrganStorage.ServerTryAdd(Added.gameObject);
```

Comment:
the problem here is it causes the loop to get an extra element while looping through

Reply:
I think we need a comment to make sure people are aware of that

Reply:
the preriodic update of organs now loops backwards to avoid that

Reply:
well what about when you remove element?

Reply:
it would runtime, I'll make a reverse for loop

Reply:
Wouldn't it still miss an element out?

Reply:
no, it gets added at the end of the list and the index used does not change